cmds/exp/dmidecode/dmidecode_test.go

```diff
@@ -54,7 +54,71 @@ func testOutput(t *testing.T, dumpFile string, args []string, expectedOutFile st
 }
 
 func TestDMIDecode(t *testing.T) {
-	testOutput(t, "UX307LA.bin", nil, "UX307LA.txt")
-	testOutput(t, "UX307LA.bin", []string{"-t", "system"}, "UX307LA.system.txt")
-	testOutput(t, "UX307LA.bin", []string{"-t", "1,131"}, "UX307LA.1_131.txt")
+	bf, _ := filepath.Glob("testdata/*.bin")
+	for _, dumpFile := range bf {
+		txtFile := strings.Replace(dumpFile, ".bin", ".txt", 1)
```

Comment:
You should probably do `strings.TrimSuffix(dumpFile, ".bin") + ".txt"` in case ".txt" appears someplace else in the file path.

Reply:
done